modules/apps/site-initializer/site-initializer-liferay-marketplace/site-initializer-liferay-marketplace-test/src/testFunctional/macros/MP2Dashboards.macro

```diff
@@ -0,0 +1,50 @@
+definition {
+
+	macro clickNewMemberButton {
+		Click(locator1 = "MP2Utils#NEW_MEMBER_BUTTON");
+	}
+
+	macro clickOnTheInvitedUser {
+		Click(
+			key_text = ${text},
+			locator1 = "MP2Utils#RESPONSIVE_TABLE");
+	}
+
+	macro inviteNewMember {
+		MP2Dashboards.clickNewMemberButton();
+
+		WaitForPageLoad();
+
+		MP2Dashboards.typeNewMemberInfoFields(
+			field = "firstName",
+			fieldInput = ${firstName});
+
+		MP2Dashboards.typeNewMemberInfoFields(
+			field = "lastName",
+			fieldInput = ${lastName});
+
+		MP2Dashboards.typeNewMemberInfoFields(
+			field = "emailAddress",
+			fieldInput = ${emailAddress});
+
+		MP2Utils.selectCheckbox(checkboxName = ${checkboxName});
+
+		if (isSet(button)) {
+			Button.click(button = "Send Invite");
+		}
+
+		Alert.viewSuccessMessage();
+	}
+
+	macro typeNewMemberInfoFields {
+		Type(
+			key_field = ${field},
+			locator1 = "MP2Dashboards#SETUP_FIELDS",
+			value1 = ${fieldInput});
+	}
+
+	macro viewNewMemberButtonNotPresent {
+		AssertElementNotPresent(locator1 = "MP2Utils#NEW_MEMBER_BUTTON");
```

Comment:
Hey @LuanaKSilva, the path NEW_MEMBER_BUTTON can be more general with a variable, because it works in other sections, e.g. in Projects in the '+ New Project' button and in Apps in the '+ New App' button.

Reply:
Same comment as above.